GitCommands/Settings/ConfigFileSettingsCache.cs

```diff
@@ -1,35 +1,37 @@
-﻿using GitCommands.Config;
+﻿using System.Diagnostics;
+using GitCommands.Config;
 using GitUIPluginInterfaces;
 
 namespace GitCommands.Settings
 {
+    [DebuggerDisplay("{_byNameMap.Count} {" + nameof(SettingsFilePath) + ",nq}")]
```

Comment:
```suggestion
    [DebuggerDisplay("Values: {_byNameMap.Count}, File: {" + nameof(SettingsFilePath) + ",nq}")]
```

Reply:
- "<<" looks better than "<-" and matches the Settings dialog.
- I can add parens for `Count`.
- The resulting strings are too long already. The labels are pretty obvious if one is debugging this code.